albam/__init__.py

```diff
@@ -63,6 +67,7 @@ def register():
     bpy.types.Material.albam_custom_properties = bpy.props.PointerProperty(type=AlbamCustomPropertiesMaterial)
     bpy.types.Mesh.albam_custom_properties = bpy.props.PointerProperty(type=AlbamCustomPropertiesMesh)
     bpy.types.Image.albam_custom_properties = bpy.props.PointerProperty(type=AlbamCustomPropertiesImage)
+    bpy.types.Scene.albam_meshes = bpy.props.PointerProperty(type=bpy.types.Object, poll=mesh_filter)
```

Comment:
It might be good to keep grouping everything under `bpy.context.scene.albam` and avoid polluting the scene attributes.
You can do it like this:

```

@blender_registry.register_blender_prop_albam(name="meshes")
class AlbamMeshes(bpy.types.PropertyGroup):
    all_meshes: bpy.props.PointerProperty(type=bpy.types.Object, poll=mesh_filter)
    # possibly add other filtered lists here in the future
   # e.g. "albam_meshes" that lists only meshes that belong to an albam asset or per-app meshes
```

Then you would access it like `bpy.context.scene.albam.meshes.all_meshes`

Reply:
But where is a better place to put it, in the tools.py?

Reply:
yeah, sounds OK, I don't think other parts would need it.